src/ui/config/config.go

```diff
@@ -240,6 +240,16 @@ func RegistryURL() (string, error) {
 	return cfg[common.RegistryURL].(string), nil
 }
 
+// RackspaceMK8SAuthURL ...
+func RackspaceMK8SAuthURL() string {
+	rackspaceMK8SAuthURL := os.Getenv("RACKSPACE_MK8S_AUTH_URL")
+	if len(rackspaceMK8SAuthURL) == 0 {
+		rackspaceMK8SAuthURL = "http://app:8080"
+	}
+
+	return rackspaceMK8SAuthURL
```

Comment:
how do you feel about doing some basic validation here? e.g. 

```
if _, err := url.ParseRequestURI(rackspaceMK8SAuthURL);  err != nil {
   
}
```

Reply:
I feel good about it. 😄